src/services/navigationBar.ts

```diff
@@ -154,46 +160,61 @@ namespace ts.NavigationBar {
             });
         }
 
-        function addTopLevelNodes(nodes: Node[], topLevelNodes: Node[]): void {
-            nodes = sortNodes(nodes);
+        // Add nodes in a single "level" of top-level nodes (as in, methods in a class.)
+        // Nodes in a single "level" are sorted together.
+        function addTopLevelNodes(nodes: Node[], higherLevel: Node[]): void {
+            const thisLevel: Node[] = [];
+            for (let node of nodes)
+                addTopLevelNode(node, thisLevel);

```

Comment:
We always use curlies in these constructs. Could you add this file to the lint sources in `Jakefile.js`?